### PR TITLE
added api endpoint to create a new order

### PR DESCRIPTION
**Pull request by LSB616**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -921,3 +921,93 @@ describe("DELETE /api/users/:user_id", () => {
...
+      .send(newOrder)
+      .expect(201)
+      .then(({ body }) => {
+        const { orders } = body;
+        expect(orders).toMatchObject(expected);
+      });
+  });
+
+
+  test("should respond with a 404 when given a malformed order", () => {
```
Comment (stormytuna) [inline __tests__/app.test.js:962]: Should say we're testing for a 404

File: models/models.users.js [hunk 2]
```diff
@@ -89,3 +89,22 @@ exports.removeUser = async (user_id) => {
 
   await User.remove({ _id: user_id})
 }
+
+
+exports.createOrder = async (user_id, order) => {
+  const expectedKeys = ['services', 'createdAt', 'fulfilledAt', 'servicedBy']
+  const receivedKeys = Object.keys(order)
+  const user = await User.findById(user_id);
```
Comment (stormytuna) [inline models/models.users.js]: If we check for 404s after the `findOneAndUpdate` we can remove this `findById` call

**Review by stormytuna — COMMENTED** [review]
Couple things and then ready to merge, great work 👍

File: __tests__/app.test.js (hunk 1 above)
Comment (LSB616) [reply __tests__/app.test.js:962]: says it in the first 5 words

**Pull request merged**